<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XMB</class>
 <widget class="QDialog" name="XMB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>541</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>846</width>
    <height>541</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>201</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="listui">
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="cursor" stdset="0">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget::item{width:40px;height:60px;}
QListWidget{background-color:#E4E4E4;}
QListWidget::item::selected:!active{background-color:rgb(98,93,255);}
QListWidget::item::selected:active{background-color:rgb(98,93,255);}
QListWidget::item::hover{background-color:rgba(50,23,255,100);}</string>
      </property>
      <item>
       <property name="text">
        <string notr="true">文      件</string>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>图      片</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <underline>false</underline>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>音      乐</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>视      频</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>其      他</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>关      于</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>651</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="mainui">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"> 
QTabBar::tab:selected, QTabBar::tab:hover
{
	color:#618BE5;
	background:transparent;
	font-family:&quot;微软雅黑&quot;;
	font-size:20px;
	
	border-bottom:2px solid #618BE5;
}

QTabBar::tab
{
	width:161px;
	height:57px;
font-family:&quot;微软雅黑&quot;;
	font-size:16px;
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Mynet">
       <attribute name="title">
        <string>我的网盘</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>0</y>
          <width>651</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="menu_layout"/>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>591</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="file_layout"/>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>190</y>
          <width>591</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="file_layout_2"/>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>330</y>
          <width>591</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="file_layout_3"/>
       </widget>
      </widget>
      <widget class="QWidget" name="trans_list">
       <attribute name="title">
        <string>传输列表</string>
       </attribute>
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>280</y>
          <width>201</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>68</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Friend">
       <attribute name="title">
        <string>好友分享</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="Find">
       <attribute name="title">
        <string>找资源</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="head_layout">
    <item row="0" column="0">
     <widget class="QToolButton" name="toolButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
